packages/yoga/src/heading/web/BackButton.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ArrowLeft } from '@gympass/yoga-icons';
+import Button from '../../Button';
+
+const BackButton = ({ onClick }) => (
+  <Button.Icon icon={ArrowLeft} inverted onClick={onClick} />
```

Comment:
I think we should add the other props to the button, otherwise it won't be possible to add the `aria-label` to the button.
```suggestion
const BackButton = ({ onClick, ...otherProps }) => (
  <Button.Icon {...otherProps} icon={ArrowLeft} inverted onClick={onClick} />
```